microsim/opencl/ramp/params.py

```diff
@@ -47,7 +47,7 @@ def __init__(self,
         """Create a simulator with the default parameters."""
         if obesity_multipliers is None:
             obesity_multipliers = [1, 1, 1, 1]
-        self.symptomatic_multiplier = 0.5

```

Comment:
I wonder if it's worth renaming this parameter to something like `symptomatic_movement_multiplier` or something like that? Just because I find it very hard to remember what this is for!

Just a thought though, totally optional!

Reply:
Also totally agree that it's worth renaming - as you know I totally forgot what it did too!